src/Swift.Runtime/src/Library/ARC.cs

```diff
@@ -0,0 +1,167 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Runtime.InteropServices;
+
+namespace Swift.Runtime;
+
+/// <summary>
+/// ARC is a class containing p/invokes for Swift reference counting and memory management.
+/// </summary>
+public static class ARC {
```

Comment:
I am wondering about the naming. I do get that AutomaticReferenceCounting is quite lengthy. Could we at least put `AutomaticReferenceCounting` in the summary string?

Reply:
> I do get that AutomaticReferenceCounting is quite length

This is not an API that will be frequently used by user code, so the long name is not a big deal. You can always do `using` for files where it is used very frequently.